SingularityBase/src/main/java/com/hubspot/singularity/SingularityUpdatePendingDeployRequest.java

```diff
@@ -0,0 +1,37 @@
+package com.hubspot.singularity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class SingularityUpdatePendingDeployRequest {
+  private final String requestId;
+  private final String deployId;
+  private final int targetActiveInstances;
+
+  public SingularityUpdatePendingDeployRequest(@JsonProperty("requestId") String requestId,
+    @JsonProperty("deployId") String deployId,
+    @JsonProperty("targetActiveInstances") int targetActiveInstances) {
+    this.requestId = requestId;
+    this.deployId = deployId;
+    this.targetActiveInstances = targetActiveInstances;
+  }
+
+  public String getRequestId() {
+    return requestId;
+  }
+
+  public String getDeployId() {
+    return deployId;
+  }
+
+  public int getTargetActiveInstances() {
+    return targetActiveInstances;
+  }
+
+  @Override public String toString() {
```

Comment:
this is a recent intellij thing that started happening, might not be a setting that the eclipse formatter file sets. i think it'd be good to keep all our `@Override`'s on their own line